index.html

```diff
@@ -1,305 +1,295 @@
-                          id="me"
-                          name="contactme"
-                          maxlength="30"
-                          placeholder="George D. Kruah"
-                          required
-                        />
-                      </li>
-                      <li>
-                        <input
-                          type="email"
-                          class="email"
-                          id="email"
-                          name="email"
-                          placeholder="dfizzkruah@gmail.com"
-                          required
-                        />
-                      </li>
-                      <li>
-                        <textarea
-                          name="message"
-                          class="message"
-                          maxlength="500"
-                          id="message"
-                          placeholder="Write your message here"
-                          required
-                        ></textarea>
-                      </li>
-                      <li>
-                        <button type="submit" name="get-in-touch">
-                          <strong>Get in Touch</strong>
-                        </button>
-                      </li>
+    </header>
+
+    <main>
+        <section class="main-content">
+
+            <div>
+                <h1 class="primary">I'm George<br>Glad to see <span>you!</span></h1>
+            </div>
+
+            <div class="profile-text">
+                <p>
+                    I'm a software developer. I can help you build a product, feature or
+                    website look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don't hesitate to
+                    contact me.
+                </p>
+            </div>
+
+            <div class="purple-text1">
+                <p>LET'S CONNECT</p>
+            </div>
+
+            <div class="social-media-icons1">
+                <ul class="social-list">
+                    <li>
+                        <a href="#"><img src="./img/twitter.svg" alt="twitter"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./img/Linkedin.svg" alt="LinkedIn"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./img/m.svg" alt="M"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./img/git.svg" alt="GIT"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./img/hn.svg" alt="HN"></a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="single-work">
+                <img src="./img/Portfolio6.png" alt="Portfolio 1">
+                <div class="holder">
+                    <h2>Tonic</h2>
```

Comment:
- [ ] Your design is awesome, great job... but there is a small issue in the positioning of your cards, they are reversed 😄 , the image should be on the left and the text on the other side as per the Figma design, please fix that, 
- [ ] ⚠️ `not only this card, but all of them have to be arranged, please take a look back to this` [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)

<img width="1033" alt="do this" src="https://user-images.githubusercontent.com/79058364/155615251-59f7a1ad-34c2-4877-8406-d35962999841.png">